livestream-compose-sample/src/main/java/io/getstream/livestream/compose/ui/LivestreamComposer.kt

```diff
@@ -1,78 +1,114 @@
             label = {
-                Row(
-                    Modifier.wrapContentWidth(),
-                    verticalAlignment = Alignment.CenterVertically
-                ) {
-                    Text(
-                        modifier = Modifier.padding(start = 4.dp),
-                        text = "Add your comments here",
-                        color = ChatTheme.colors.textLowEmphasis
-                    )
-                }
-            }
-        )
-        val isInputValid = composerViewModel.input.isNotEmpty()
-        IconButton(
-            enabled = isInputValid,
-            content = {
-                Icon(
-                    imageVector = Icons.Default.Send,
-                    contentDescription = stringResource(id = R.string.stream_compose_send_message),
-                    tint = if (isInputValid) ChatTheme.colors.primaryAccent else ChatTheme.colors.textLowEmphasis
+                Text(
+                    modifier = Modifier
+                        .padding(start = 4.dp)
+                        .wrapContentWidth()
+                        .align(Alignment.CenterVertically),
+                    text = stringResource(id = R.string.composer_hint),
+                    color = ChatTheme.colors.textLowEmphasis
                 )
-            },
-            onClick = {
-                if (isInputValid) {
-                    composerViewModel.sendMessage(composerViewModel.input)
-                }
             }
         )
+        Column(
+            modifier = Modifier
+                .padding(start = 4.dp)
+                .clip(RoundedCornerShape(8.dp))
+                .background(context.cardBackground())
+                .width(40.dp)
+                .height(40.dp)
+                .clickable(
+                    interactionSource = remember { MutableInteractionSource() },
+                    indication = rememberRipple(bounded = false),
+                    onClick = {
+                        composerViewModel.sendMessage(
+                            Message(
+                                cid = channelId,
+                                text = composerViewModel.input
+                            )
+                        )
+                    }
+                ),
+        ) {
+            Icon(
+                modifier = Modifier
+                    .weight(1f)
+                    .align(Alignment.CenterHorizontally),
+                painter = painterResource(R.drawable.ic_arrow),
+                contentDescription = stringResource(id = R.string.stream_compose_send_message),
+                tint = ChatTheme.colors.textLowEmphasis
+            )
+            Text(
+                modifier = Modifier
+                    .weight(1f)
+                    .padding(3.dp)
+                    .align(Alignment.CenterHorizontally),
+                fontSize = 11.sp,
+                color = ChatTheme.colors.textLowEmphasis,
+                text = "280"
```

Comment:
Why is this hardcoded? If we're not supporting the message limits for the initial version of the livestream app, then it doesn't make sense to include this and hardcode a number.

We should probably just use a basic send `Icon` instead for the initial version. We discussed and agreed that this isn't important for the initial version of the app!

Reply:
Removed  - the reason to keep it was be as to design as possible
In same context i should also not add `info` card as is then ( we dont have supporting functionality yet) 
![screen6](https://user-images.githubusercontent.com/1392571/129673930-4193a9fd-2fc9-4cc1-bb38-61859792bec1.png)